### PR TITLE
feat(image_projection_based_fusion): add trust distance to fusion

### PR DESCRIPTION
**Pull request by kaancolak**
Signed-off-by: Kaan Colak <kaancolak95@gmail.com>

## Description

#2659 

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [X] I've confirmed the [contribution guidelines].
- [X] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by kaancolak** [comment]
Hi @yukkysaito -san , Could you review this PR? 

Sorry for the delay, I was working on other tasks.

**Review by yukkysaito — APPROVED** [review]
LGTM

File: perception/image_projection_based_fusion/include/image_projection_based_fusion/roi_cluster_fusion/node.hpp [hunk 1]
```diff
@@ -44,7 +44,9 @@ class RoiClusterFusionNode
   bool use_cluster_semantic_type_{false};
   float iou_threshold_{0.0f};
   bool remove_unknown_;
+  float thrust_distance_;
```
Comment (miursh) [inline perception/image_projection_based_fusion/include/image_projection_based_fusion/roi_cluster_fusion/node.hpp:47]: [Q]
What does "thrust" mean? Is it like upper-limit or maximum? Just a typo of "TRUST"?

Comment (kaancolak) [reply perception/image_projection_based_fusion/include/image_projection_based_fusion/roi_cluster_fusion/node.hpp]: Sorry, @miursh. Fixed it,  it's just a typo.

**Review by miursh — APPROVED** [review]
LGTM

**Review by yukke42 — APPROVED** [review]
LGTM

**Pull request merged**